### PR TITLE
ui: [Backport/1.12.x] Add documentation link to the Server Fault Tolerance panel (#12792)

### PR DESCRIPTION
**Pull request by evrowe**
Backport of #12792

Adds a 'Learn more' link to the top of the Server Fault Tolerance panel. See original PR for more details.

no changelog due to base not being `main`

### CODE REVIEW

File: ui/packages/consul-ui/translations/routes/en-us.yaml
Comment (johncowen) [inline ui/packages/consul-ui/translations/routes/en-us.yaml:45]: Didn't I see another PR go in somewhere to fix this up (I don't think the `docs` bit should be there). I'm not totally sure how this will merge down, so maybe its worth merging this anyway and then following up with a quick once over and correction once all these backports are in. I'll leave it up to you how you want to do this but we need to finish up without this extra `docs` in there

**Review by johncowen — APPROVED** [review]
:shipit: I made a little note on that extra `docs`, if it was me I'd just get all these backports in and then give everything a once over once its all where it should be, up to you how you want to do it though.

File: ui/packages/consul-ui/translations/routes/en-us.yaml
Comment (evrowe) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:45]: This might be the result of a bad merge conflict resolution from me. The original PR (#12792) has the changes that were submitted to make this change for the release. I'll double check this to ensure it reflects the intended change.

Comment (jkirschner-hashicorp) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:45]: This was the PR to fix the `docs/docs` broken link here: https://github.com/hashicorp/consul/pull/12815

File: ui/packages/consul-ui/translations/routes/en-us.yaml
Comment (jkirschner-hashicorp) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:9]: Actually, this is also the wrong link. It should be:

```suggestion
          <a href="{CONSUL_DOCS_URL}/architecture/improving-consul-resilience#strategies-to-increase-fault-tolerance](https://www.consul.io/docs/architecture/improving-consul-resilience#strategies-to-increase-fault-tolerance)" target="_blank" rel="noopener noreferrer">Learn how to improve fault tolerance</a>
```

Comment (jkirschner-hashicorp) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:9]: ... Just now realizing this is a backport. So this fix would also need to apply to `main`: https://github.com/hashicorp/consul/pull/12792

Comment (jkirschner-hashicorp) [reply ui/packages/consul-ui/translations/routes/en-us.yaml:9]: Let me know if I should approve the backport, and instead we'll address this comment in a subsequent PR. That seems like the right course of action.

**Review by jkirschner-hashicorp — APPROVED** [review]
lgtm

**Comment by jkirschner-hashicorp** [comment]
I assume the link/CTA text change will need to happen on `main` also?

File: ui/packages/consul-ui/translations/routes/en-us.yaml
Comment (jkirschner-hashicorp) [inline ui/packages/consul-ui/translations/routes/en-us.yaml:45]:
```suggestion
              <a href="{CONSUL_DOCS_URL}/enterprise/license/faq#q-how-can-i-renew-a-license" target="_blank" rel="noopener noreferrer">
```

**Review by jkirschner-hashicorp — CHANGES_REQUESTED** [review]
One small fix to a link (to match the #12815 fix)

**Comment by evrowe** [comment]
> I assume the link/CTA text change will need to happen on main also? I can submit that PR if needed

@jkirschner-hashicorp I think this may already be on `main` as a result of #12792 but please double check this

**Comment by jkirschner-hashicorp** [comment]
@evrowe : it looks like #12792 on `main` uses a different link than is used here. (The one used here is correct.)

After this is merged, I'll fix the link on main and submit a PR. Thanks!

**Review by jkirschner-hashicorp — APPROVED** [review]
LGTM

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/666687.

**Pull request merged**
